packages/frontend/src/components/common/PivotTable/index.tsx

```diff
@@ -46,15 +69,14 @@ const PivotTable: FC<PivotTableProps> = ({
         >
             <thead>
                 {data.headerValueTypes.map(
-                    (_headerValueType, headerValueTypeIndex) => {
```

Comment:
from here till the end of the file is just renaming arguments to make it clear which one is which (row index or column index)